database/postgres/repo_list_test.go

```diff
@@ -148,7 +148,81 @@ func TestPostgres_Client_GetOrgRepoList(t *testing.T) {
 
 	// run tests
 	for _, test := range tests {
-		got, err := _database.GetOrgRepoList("foo", 1, 10)
+		got, err := _database.GetOrgRepoList("foo", []string{""}, 1, 10)
```

Comment:
similar to previous comment, we want a test where the exclude is populated as well, right?

Reply:
I'll add a test, however we are mocking the result-set, so not sure it accomplishes anything.

Any plans on setting up a Postgres instance for tests?

Reply:
Not sure I follow. We create a test database and seed it with data and then mock the response we expect to get given the test.

Reply:
that's fair. i was thinking about sqlite, i think :)